vmu.py

```diff
@@ -115,3 +90,86 @@ def canonical_src_string(sources):
     result = re.sub(r',', ' + ', result)
     result = re.sub(r'^(\d+\.\d+)(.*-> )(?!\d)', '\\1\\2\\1 ', result) # Duplicate release series, when needed
     return result.strip()
+
+class ParamError(Exception):
+    """Exception for errors in parameter files"""
+    pass
+
+class PackageSet(object):
+    """A class for comparing package sets as specified in VMU test parameters
+
+    Instance parameters:
+
+    label: string describing the package set, used for reporting
+    packages: list of packages as strings
+    selinux: boolean to turn on/off SELinux enforcing mode (default: False)
+    java: boolean to pre-install OSG java packages (default: True)
+
+    PackageSets are equal if their 'packages' lists are the same. If 'packages'
+    lists are equal but 'label' is not, ParamError is raised.
```

Comment:
As mentioned in the docstring, note that a `PackageSet` object is really a package list -- two `PackageSet` objects with the same set of packages listed _in a different order_ are distinct.  (Not that this is a problem, it just might make the name of the class slightly misleading.)